### PR TITLE
Qbank api

### PR DESCRIPTION
**Pull request by fugue88**

### CODE REVIEW

File: lambda/proxy/QBankFetcher.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+"use strict";
```
Comment (fugue88) [inline lambda/proxy/QBankFetcher.js:1]: This file came from MIT, but we're not using it.

File: lambda/proxy/QBankSignature.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
+'use strict';
```
Comment (fugue88) [inline lambda/proxy/QBankSignature.js:1]: This file came from MIT, but we're not using it.

File: lambda/proxy/Util.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+"use strict";
```
Comment (fugue88) [inline lambda/proxy/Util.js:1]: This file came from MIT, but we're not using it.

**Comment by BenjaminCall** [comment]
This is excellent code.

**Pull request merged**